firefox-ios/Client/Application/ImageIdentifiers.swift

```diff
@@ -73,5 +73,4 @@ public struct ImageIdentifiers {
     public static let zoomIn = "menu-ZoomIn"
     public static let customizeHomepage = "editLarge"
     public static let help = "helpCircleLarge"
-    public static let share = "Share"
```

Comment:
Is this intentional? wondering if there might be a test affected?

Reply:
> Is this intentional? wondering if there might be a test affected?

This is intentional. It is an overdue update based on @thatswinnie request from this PR: https://github.com/mozilla-mobile/firefox-ios/pull/18301